docs/api-guide.md

```diff
@@ -0,0 +1,718 @@
+boolean ifNotExists = true;
+admin.createNamespace("ns", ifNotExists);
+
+// Create a namespace with options
+Map<String, String> options = ...;
+admin.createNamespace("ns", options);
+```
+
+#### Create Options
+
+In the creation operations (creating a namespace, creating a table, etc.), you can specify options that are maps of option name and value (`Map<String, String>`).
+With the options, we can set storage adapter specific configurations 
+
+Currently, we can set the following options for the storage adapters:
+
+For Cosmos DB:
+
+| name       | value                              | default |
+| ---------- | ---------------------------------- | ------- |
+| ru         | Base resource unit                 | 400     |
+| no-scaling | Disable auto-scaling for Cosmos DB | false   |
+
+For DynamoDB:
+
+| name       | value                                  | default |
+| ---------- | -------------------------------------- | ------- |
+| no-scaling | Disable auto-scaling for DynamoDB      | false   |
+| no-backup  | Disable continuous backup for DynamoDB | false   |
+| ru         | Base resource unit                     | 10      |
+
+For Cassandra:
+
+| name                 | value                                                                                 | default          |
+| -------------------- | ------------------------------------------------------------------------------------- | ---------------- |
+| replication-strategy | Cassandra replication strategy, must be `SimpleStrategy` or `NetworkTopologyStrategy` | `SimpleStrategy` |
+| compaction-strategy  | Cassandra compaction strategy, must be `LCS`, `STCS` or `TWCS`                        | `STCS`           |
+| replication-factor   | Cassandra replication factor                                                          | 1                |
+
+
+### Create a table
+
+Next, we will discuss table creation.
+
+You firstly need to create the TaleMetadata as follows:
+
+```java
+// Define a table metadata
+TableMetadata tableMetadata =
+    TableMetadata.newBuilder()
+        .addColumn("c1", DataType.INT)
+        .addColumn("c2", DataType.TEXT)
+        .addColumn("c3", DataType.BIGINT)
+        .addColumn("c4", DataType.FLOAT)
+        .addColumn("c5", DataType.DOUBLE)
+        .addPartitionKey("c1")
+        .addClusteringKey("c2", Scan.Ordering.Order.DESC)
+        .addClusteringKey("c3", Scan.Ordering.Order.ASC)
+        .addSecondaryIndex("c4")
+        .build();
+```
+
+Here you define columns, a partition key, a clustering key including clustering orders, and secondary indexes of a table.
+
+Please see [Scalar DB design document - Data Model](design.md#Data Model) for the details of the Scalar DB Data Model.
+
+And then, you can create a table as follows: 
+
+```java
+// Create a table "ns.tbl"
+admin.createTable("ns", "tbl", tableMetadata);
+
+// Create a table if not exists
+boolean ifNotExists = true;
+admin.createTable("ns", "tbl", tableMetadata, ifNotExists);
```

Comment:
```suggestion
// Create a table "ns.tbl". It will throw an exception if the table already exists
admin.createTable("ns", "tbl", tableMetadata);

// Create a table only if it does not already exist
boolean ifNotExists = true;
admin.createTable("ns", "tbl", tableMetadata, ifNotExists);
```